### PR TITLE
Some proof-reading: chapters 1 to 3

### PR DESCRIPTION
**Pull request by cormullion**
re #137 

I made some minor copy edits to the first three chapters, hope I haven't disturbed the book's syntax.

I'll admit that a fair few of these are likely matters of personal taste :) Sometimes it's just an attempt at rephrasing to make the sentences flow a bit more smoothly to my ears.

### CODE REVIEW

File: contents/why_julia.md
Comment (rikhuijzer) [inline contents/why_julia.md:228]: Awesome. I've also noticed that this word was off, but never spotted the fix

**Review by rikhuijzer — APPROVED** [review]
Thanks a lot @cormullion. It reads much better now! All three of us are not native speakers so it's very nice that you have took the time to take a look.

I also just now noticed that you add a lot of Oxford commas. We didn't really talk about it as co-authors, so I guess that we'll go with Oxford commas indeed.

I'll wait for a review from @storopoli for this PR too since he was the main author of a lot of the text that you reviewed. I completely approve merging this :+1:

**Comment by lazarusA** [comment]
@cormullion thanks!

**Review by storopoli — APPROVED** [review]
Thank you @cormullion!

**Pull request merged**
